paywall/src/static/adremover/adRemoverUtils.js

```diff
@@ -39,6 +39,7 @@ function setupPaywallConfig(window) {
   //   ...,
   // ]
   var locks = JSON.parse(us.searchParams.get('locks'))
+  var unlockUserAccounts = !!us.searchParams.get('unlockUserAccounts')
```

Comment:
How would you feel about renaming this variable? `unlockUserAccounts` doesn't intuitively seem like a boolean to me.

Reply:
I'd feel great. However, it is currently in master as `unlockUserAccounts`. Let's talk with @julien51 about ideas?